internal/sigstore/client.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2022 Red Hat, Inc.
```

Comment:
This is a direct copy from `internal/evaluation_target/application_snapshot_image/client.go`. This was needed to avoid circular imports. This PR needs to be improved to delete that one in favor of this. I consider it a blocker.